include/doap-geenrator.html

```diff
@@ -0,0 +1,27 @@
+<html>
+    <head>
+        <title>DOAP Generator</title>
+        <script>
+            function generate(){
+                            console.log("Generate");
+                        }
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <form id="project">
+                <label for="name">Project Name</label>
```

Comment:
All this just placeholders for now?

Reply:
This html document is superfluous now so have removed